ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.js

```diff
@@ -38,6 +40,13 @@ export default class MessagesList extends Component {
     }
   }
 
+  get hasSomeActiveModals() {
+    return (
+      (this.args.messages || []).some((message) => message.type === 'modal' && message.active) &&
+      this.args.message.type === 'modal'
+    );
+  }
```

Comment:
instead of the `||` conditional here, could we return false if `!this.args.messages`? It might read a little more clearly that way 
```suggestion
if (!this.args.messages) return false
    return (this.args.messages.some((message) => message.type === 'modal' && message.active) &&
      this.args.message.type === 'modal'
    );
  }
```